File: EldenRing_API/demo/src/main/java/com/example/demo/Attribute.java
package com.example.demo;

import lombok.Data;

public class Attribute
{
    public String getName() {
        return name;
    }

    public Integer getAmount() {
        return amount;
    }

    public void setName(String name) {
        this.name = name;
    }

    public void setAmount(Integer amount) {
        this.amount = amount;
    }

    private String name;
    private Integer amount;

    public Attribute(String name, Integer amount) {
        this.name = name;
        this.amount = amount;
    }
}

File: EldenRing_WebApp/EldenRing_WebApp/Data/Attribute.cs
namespace EldenRing_WebApp.Data
{
    public class Attribute
    {
        public string name { get; set; }
        public int amount { get; set; }

        public Attribute(string name, int amount)
        {
            this.name = name;
            this.amount = amount;
        }
    }
}
